Dockerfile

```diff
@@ -0,0 +1,57 @@
+# (C) Copyright 2018 The o2r project. https://o2r.info
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM alpine:3.6
+
+# Add Alpine mirrors, replacing default repositories with edge ones, based on https://github.com/jfloff/alpine-python/blob/master/3.4/Dockerfile
+RUN echo \
+  && echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" > /etc/apk/repositories \
+  && echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories \
+  && echo "http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories
+
+RUN apk add --no-cache \
+    nodejs \
+    dumb-init \
+    nodejs-npm \
+  && rm -rf /var/cache
+
+WORKDIR /guestlister
+COPY config config
+COPY lib lib
+COPY css css
+COPY views views
+COPY app.js app.js
+COPY package.json package.json
+RUN npm install --production
```

Comment:
Tipp: Do `copy package.json` and `npm install` before copying all other files, it improves performance when testing because Docker can cache the installation.